src/library/scala/collection/immutable/Set.scala

```diff
@@ -71,7 +71,7 @@ object Set extends ImmutableSetFactory[Set] {
     def + (elem: Any): Set[Any] = new Set1(elem)
     def - (elem: Any): Set[Any] = this
     def iterator: Iterator[Any] = Iterator.empty
-    override def foreach[U](f: Any =>  U): Unit = {}
+    override def foreach[U](f: Any => U): Unit = {}
```

Comment:
RHS changed to `()`.

FTR occurrences that remain

Style  | Count 
------------ | -------------:
Unit = {} | 35
Unit = () | 202